libs/motr/motr_core_k8s_lib.py

```diff
@@ -360,13 +360,16 @@ def diff_cmd(self, file1, file2, node):
         :file2: second file
         :node: compare files on which node
         """
-
+        diff_utils_install = common_cmd.INSTALL_DIFF_UTIL
         cmd = common_cmd.DIFF.format(file1, file2)
-        resp = self.node_obj.send_k8s_cmd(operation="exec", pod=self.node_pod_dict[node],
```

Comment:
In general I can see a lot of optimization can be done at `commands.py`  but will differ that review for sometime later in the housekeeping.

Reply:
sure.

Reply:
sure.